cl/lib/utils.py

```diff
@@ -118,3 +118,75 @@ def human_sort(
         sorter = lambda item: [convert(c) for c in re.split("([0-9]+)", item)]
 
     return sorted(unordered_list, key=sorter)
+
+
+def microservice(
+    service: str,
+    method: str = "POST",
+    doc: Any = None,
+    file: bytes = None,
+    filename: str = None,
+    filepath: str = None,
+    data=None,
+    params=None,
+) -> Response:
+    """Call a Microservice endpoint
+
+    This is a helper utility to call our microservices.  To see a list of Endpoints
+    check out the settings file cl/settings/public.py.
+
+    Because of the various ways our db is setup we have a few different params we use
+    in this function.
+
+    :param service: The service to call
+    :param method: The method to use (defaults to POST)
+    :param doc: The document as a db object
+    :param file: The file as a byte array
+    :param filename: The filename of the file
```

Comment:
If this is used only as a placeholder for "extension", we should definitely call it the extension and pass it as such.

Reply:
noted.  ill get back to you on that.